Docs/install/install_docker.rst

```diff
@@ -37,132 +37,255 @@
 .. _installation-docker:
 
 ##############################
-AIMET Installation in Docker
+AIMET Docker install and setup
 ##############################
 
-This page provides instructions to install AIMET package inside a development docker container.
+This page describes how to install AIMET in a development Docker container.
 
-Set variant
-~~~~~~~~~~~
-Set the `<variant_string>` to ONE of the following depending on your desired variant
-    #. For the PyTorch 2.1 GPU variant, use `torch-gpu`
-    #. For the PyTorch 2.1 CPU variant, use `torch-cpu`
-    #. For the PyTorch 1.13 GPU variant, use `torch-gpu-pt113`
-    #. For the TensorFlow GPU variant, use `tf-gpu`
-    #. For the TensorFlow CPU variant, use `tf-cpu`
-    #. For the ONNX GPU variant, use `onnx-gpu`
-    #. For the ONNX CPU variant, use `onnx-cpu`
+Prerequisites
+=============
+
+Ensure that you have Docker installed.
```

Comment:
If we're adding the pre-requisite section, it is better to be more specific like this:
- Install Docker from https://docs.docker.com/engine/install/ubuntu/
- NVIDIA Container Toolkit from https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html (_required_ for GPU variants)

Reply:
Added.